### ISSUE TITLE
Parse "ALTER SYSTEM" in Postgres

### ISSUE THREAD

**Issue by nuno-agostinho** [sqlfluff/sqlfluff#7446]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

SQLFluff 4.0.0 reports a parsing error for valid Postgres `ALTER SYSTEM` statements.

### Expected Behaviour

Correctly parse `ALTER SYSTEM` commands in Postgres.

Maybe this could be addressed similarly to how `ALTER SYSTEM` is handled in Exasol: https://github.com/sqlfluff/sqlfluff/blob/e16ad28a9741bef7fe8aeff7a0e4b1ca3ef7a5a7/src/sqlfluff/dialects/dialect_exasol.py#L2842-L2853

### Observed Behaviour

SQLFluff cannot parse it:
```
L:   4 | P:   1 |  PRS | Line 4, Position 1: Found unparsable section: "ALTER
                       | SYSTEM SET max_wal_size = '2GB';\nS..."
WARNING: Parsing errors found and dialect is set to 'postgres'. Have you configured your dialect correctly?
All Finished 📜 🎉!
```

### How to reproduce

For example, this line in my Postgres initialisation script:
```sql
-- Increase max_wal_size to reduce how often checkpoints occur
ALTER SYSTEM SET max_wal_size = '2GB';
```

### Dialect

postgres

### Version

SQLFluff 4.0.0
Python 3.14.2

### Configuration

[sqlfluff]
dialect = postgres

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
